ztp/source-policy-crs/TunedPerformancePatch.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: performance-patch
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+    - name: performance-patch
+      data: |
+        [main]
+        summary=Configuration changes profile inherited from performance created tuned
+        include=openshift-node-performance-performance
```

Comment:
This will need to match the name of the performance profile

Reply:
okay, will move it then to the site-specific template.